### ISSUE TITLE
AP on COCO

### ISSUE THREAD

**Issue by developer0hye** [MarkAny-Vision-AI/CenterNeXt#13]
commit sha: 58c1a02835fc60dd6ce7533aba31972cba1f4592
ap: 26

**Comment by developer0hye** [comment]
@yjh0410 Do you think the normalization method is important to achieve high detection accuracy?

**Comment by yjh0410** [comment]
@developer0hye 
If the "normalization method" you said is the method to normalize input image during preprocessing, yes, I think it is one of the key factors for advanced detectors.

**Comment by developer0hye** [comment]
@yjh0410 

Yes, that is what I mean. I didn't apply the input normalization technique because it seems one of the best detectors, YOLOv5, doesn't apply this technique to their pre-processing method.

**Comment by yjh0410** [comment]
@developer0hye 
Given an input image whose pixel value is range from 0 to 255, it is just be divided by 255 without pixel mean or pixel std. Am I understanding it right?

**Comment by developer0hye** [comment]
@yjh0410 
Yes, you are right. Image is divided by 255.

**Comment by yjh0410** [comment]
@developer0hye 
So, you didn't do any normalization method, and just convert it to tensor [C, H, W] and divide it by 255. 

I noticed that your ResNet backbone is from timm. As far as I know, the imagenet pretrained model in timm all use imagenet pixel mean=[0.485, ...] and pixel std=[0.229, ...]. If you load their pretrained weight, I think you'd better to use the imagenet pixel mean and std. Of course, if the epoch is set as very big number like 300, the negative effect might be ignored. I noticed that you set epoch to 70, it might be not enough to eliminate this negative effect. 

On the other hand, I think 70 epoch might be small for COCO dataset. Maybe 140 epoch is appropriate. How do you think?

**Comment by developer0hye** [comment]
@yjh0410 

I agree with you. 

Okay, I will apply normalization to our model! Thanks!

Now, I am training a model with a longer training schedule(140 epochs)

**Comment by yjh0410** [comment]
@developer0hye 
Looking forward to good news ！

**Comment by developer0hye** [comment]
@yjh0410 

Normalization using ImageNet statistical didn't improve performance.

**Comment by yjh0410** [comment]
@developer0hye 

I've recently been rethinking the importance of image normalization. I later thought that when using large epochs like 140 epochs or even 300 epochs to train model, normalization becomes less important. Training with so many epochs is enough to remove the variance introduced by normalization.

I notice that you try the augmentation used in YOLOv5, but the performance drops.  I want to know which augmentation you try. Mosaic and Mixup?

**Comment by developer0hye** [comment]
@yjh0410 

I only applied [random_perspective](https://github.com/MarkAny-Vision-AI/CenterNeXt/blob/yolov5-augmentation/utils/data/transforms.py#L399-L485) augmentation. I didn't applied Mosaic and Mixup because original CenterNet didn't use it as augmentation.


https://github.com/MarkAny-Vision-AI/CenterNeXt/blob/yolov5-augmentation/utils/data/dataset.py#L69
